pkg/pillar/types/diskmetrics.go

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2020. Zededa, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package types
+
+import (
+	"github.com/lf-edge/eve/pkg/pillar/base"
+)
+
+//ReportDiskPaths Report disk usage for these paths
+var ReportDiskPaths = []string{
+	"/",
+	IdentityDirname,
+	PersistDir,
+}
+
+//ReportDirPaths  Report directory usage for these paths
+var ReportDirPaths = []string{
+	PersistDir + "/downloads", // XXX old to be removed
+	PersistDir + "/img",       // XXX old to be removed
+	PersistDir + "/tmp",
+	PersistDir + "/log",
+	PersistDir + "/rsyslog",
+	PersistDir + "/config",
+	PersistDir + "/status",
+	PersistDir + "/certs",
+	PersistDir + "/checkpoint",
+}
+
+//AppPersistPaths  Application-related files live here
+// XXX do we need to exclude the ContentTrees used for eve image update?
+// If so how do we tell them apart
+var AppPersistPaths = []string{
+	VolumeEncryptedDirName,
+	VolumeClearDirName,
+	SealedDirName + "/downloader",
+	SealedDirName + "/verifier",
+}
+
+//DiskMetricStatus holds metrics data per disk
+type DiskMetricStatus struct {
```

Comment:
Please name this DiskMetric; Status makes sense as a response to Config but this is not that.

Reply:
Changed.